plugins/woocommerce/src/Internal/CLI/Migrator/Core/ProductsController.php

```
@@ -860,6 +862,37 @@ private function display_migration_summary(): void {
 		WP_CLI::line( '' );
 	}
 
+	/**
+	 * Log session time metrics using the tracker data.
+	 *
+	 * @param array $final_stats Final migration statistics.
+	 */
+	private function log_session_time_metrics( array $final_stats ): void {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$final_stats'. (undefined)

(UnusedFormalParameter)